controllers/endpoints-controller.js

```diff
@@ -0,0 +1,13 @@
+const { readFile } = require("fs/promises");
+
+exports.getEndpoints = (req, res, next) => {
+  readFile("./endpoints.json")
+    .then((file) => {
+      const endPoints = JSON.parse(file);
+      delete endPoints[Object.keys(endPoints)[0]];
+      res.status(200).send({ endPoints });
```

Comment:
You can require in the JSON without an fs.readfile here. 
But what you've done is not wrong :)